docs/tutorials/rootless_tutorial.md

```diff
@@ -87,11 +104,11 @@ The majority of the work necessary to run Podman in a rootless environment is on
 
 Once the Administrator has completed the setup on the machine and then the configurations for the user in /etc/subuid and /etc/subgid, the user can just start using any Podman command that they wish.
 
-### User Configuration Files.
+### User Configuration Files
 
-The Podman configuration files for root reside in /usr/share/containers with overrides in /etc/containers.  In the rootless environment they reside in ${XDG\_CONFIG\_HOME}/containers and are owned by each individual user.  The main files are libpod.conf and storage.conf and the user can modify these files as they wish.
+The Podman configuration files for root reside in `/usr/share/containers` with overrides in `/etc/containers`.  In the rootless environment they reside in `${XDG_CONFIG_HOME}/containers` (usually `~/.config/containers`) and are owned by each individual user.  The main files are `libpod.conf` and `storage.conf` and the user can modify these files as they wish.
 
-The default authorization file used by the `podman login` and `podman logout` commands reside in ${XDG\_RUNTIME\_DIR}/containers/auth.json.
+The default authorization file used by the `podman login` and `podman logout` commands reside in `${XDG_RUNTIME_DIR}/containers/auth.json`.
```

Comment:
I don't recall why, but at one point we needed to have the backslashes in these environment variables.  Hopefully what ever required that is no longer an issue.

Reply:
@TomSweeneyRedHat: the `_` character needs to be escaped in Markdown so as not trigger rendering of italics.

But that only happens if the underscore is not inside a preformatted section (i.e. one wrapped by backtics). Since I'm wrapping the variable in backtics, the backslashes can go.

Reply:
Ah!  Thanks for the explanation!